ODIN_II/regression_test/benchmark/operators/pca_1_always_block_4_if_input

```diff
@@ -0,0 +1,30 @@
+GLOBAL_SIM_BASE_CLK reset set set1 set2 a
+0 0 0 0 0 0
```

Comment:
Again - only case for deassign. What happens when we deassign after different assigns have occurred?